packages/phone-number-privacy/common/src/interfaces/requests.ts

```diff
@@ -15,6 +15,11 @@ import {
 } from '@celo/utils/lib/sign-typed-data-utils'
 import { verifyEIP712TypedDataSigner } from '@celo/utils/lib/signatureUtils'
 
+export enum KeyName {
+  phoneNumberPrivacy = 'phoneNumberPrivacy',
+  domains = 'domains',
+}
+
```

Comment:
IIUC, these are the defaults used by the signer. Since it is only used in that package, and the client will not know what name the signer uses for their keys, I'd recommend this be moved into the signer package.

On a less important note, I am trying to make our enum conventions more consistent. We use CapitalizedCamelCase and CAPITALIZED_SNAKE_CASE in different places, but I think the latter is the wider used standard.
```suggestion
export enum DefaultKeyName {
  PHONE_NUMBER_PRIVACY = 'phoneNumberPrivacy',
  DOMAINS = 'domains',
}
```